crates/agglayer-contracts/src/lib.rs

```diff
@@ -100,6 +123,12 @@ where
                 ));
             }
 
+            debug!(
+                "Retrieved the default L1 Info Tree entry. leaf_count: {}, root: {}",
```

Comment:
You could use structured logging on new log lines. We should also probably agree and make some dictionary about how we consistently reference common terms and entities in the comments/docs. E.g. is it `l1 info tree1 or `L1 Info Tree` or `l1_info_tree`. Same for `Aggsender` vs `AggSender` vs `aggsender` vs `agg-sender`

Reply:
That's a backport PR, I would prefer not to introduce too many changes from the original PR